### ISSUE TITLE
Possibility to crop images

### ISSUE THREAD

**Issue by mrnfrancesco** [pwndoc/pwndoc#101]
Hi,
it could be great to be able to crop images directly on the pwndoc's editor.

**Comment by yeln4ts** [comment]
yes it would be great. It's not the priority right now but hopefully one day

**Comment by noraj** [comment]
[writehat](https://github.com/blacklanternsecurity/writehat) has support for

> Crop, annotate, caption, and upload images

It may give you architecture ideas.
